From 36f5ace2db7fc43796107b2da9874e4c4bbc623e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Tue, 26 Aug 2014 21:14:11 -0400
Subject: [PATCH] man: reword sd-hibernate-resume description and add link

"each device" was suggesting that this service might be instantiated
multiple times. "hibernation resume" was too jargon-y.
---
 man/systemd-hibernate-resume@.service.xml | 11 ++++++-----
 1 file changed, 6 insertions(+), 5 deletions(-)

diff --git a/man/systemd-hibernate-resume@.service.xml b/man/systemd-hibernate-resume@.service.xml
index 9b188b0d96..30bfd88101 100644
--- a/man/systemd-hibernate-resume@.service.xml
+++ b/man/systemd-hibernate-resume@.service.xml
@@ -54,13 +54,14 @@
         <refsect1>
                 <title>Description</title>
 
-                <para><filename>systemd-hibernate-resume@.service</filename> is a
-                service that initiates hibernation resume from a device
-                containing the resume image. It is instantiated for each
-                device that is configured for resuming from.</para>
+                <para><filename>systemd-hibernate-resume@.service</filename>
+                initiates the resume from hibernation. It is
+                instantiated with the device to resume from as the
+                template argument.</para>
 
                 <para><filename>systemd-hibernate-resume</filename> only supports
-                the in-kernel hibernation implementation, known as swsusp.
+                the in-kernel hibernation implementation, known as
+                <ulink url="https://www.kernel.org/doc/Documentation/power/swsusp.txt">swsusp</ulink>.
                 Internally, it works by writing the major:minor of specified
                 device node to <filename>/sys/power/resume</filename>.</para>
 
